projects/ccf-eui/src/app/modules/ontology-exploration/ontology-selection/ontology-selection.component.ts

```diff
@@ -58,9 +58,11 @@ export class OntologySelectionComponent implements OnChanges {
 
   currentNodes: string[];
 
-  menuOptions;
+  biomarkerMenuOptions;
```

Comment:
Why rename?

Reply:
Just to specify that menu options are specific to biomarker.